src/main/main.ml

```diff
@@ -107,37 +111,39 @@ let int_arg r arg =
       | 'm' -> multiplier 60.
       | 'h' -> multiplier 3600.
       | 'd' -> multiplier 86400.
-      | '0'..'9' -> r := float_of_string arg
+      | '0' .. '9' -> r := float_of_string arg
       | _ -> raise (Arg.Bad "bad numeric argument")
-    with Failure _ -> raise (Arg.Bad "bad numeric argument")
+    with Failure _ -> raise (Arg.Bad "bad numeric argument") )
 
-let setup_gc_stat () =
-  at_exit (fun () ->
-      Gc.print_stat stdout;
-    )
+let setup_gc_stat () = at_exit (fun () -> Gc.print_stat stdout)
 
-let input_file = fun s -> file := s
+let input_file s = file := s
 
 let usage = "Usage : main [options] <file>"
-let argspec = Arg.align [
-    "-bt", Arg.Unit (fun () -> Printexc.record_backtrace true),
-    " Enable stack traces";
-    "-cnf", Arg.Set p_cnf,
-    " Prints the cnf used.";
-    "-check", Arg.Set p_check,
-    " Build, check and print the proof (if output is set), if unsat";
-    "-dot", Arg.Set_string p_dot_proof,
-    " If provided, print the dot proof in the given file";
-    "-gc", Arg.Unit setup_gc_stat,
-    " Outputs statistics about the GC";
-    "-size", Arg.String (int_arg size_limit),
-    "<s>[kMGT] Sets the size limit for the sat solver";
-    "-time", Arg.String (int_arg time_limit),
-    "<t>[smhd] Sets the time limit for the sat solver";
-    "-v", Arg.Int (fun i -> Log.set_debug i),
-    "<lvl> Sets the debug verbose level";
-    "-no-proof", Arg.Set no_proof, " disable proof logging";
-  ]
+
+let argspec =
+  Arg.align
+    [ ( "-bt",
+        Arg.Unit (fun () -> Printexc.record_backtrace true),
+        " Enable stack traces" );
+      "-cnf", Arg.Set p_cnf, " Prints the cnf used.";
```

Comment:
The absence of parentheses for these one-line tuples makes this quite hard to read I find. Is there an option to add the parens back ?

Reply:
yes, probably.